src/main/java/Exceptions/ChatAlreadyExistsException.java

```diff
@@ -0,0 +1,20 @@
+package Exceptions;
+
+import java.lang.RuntimeException;
+import org.springframework.http.HttpStatus;
+
+public class ChatAlreadyExistsException extends RuntimeException {
+
+    public static HttpStatus httpStatus = HttpStatus.CONFLICT;
```

Comment:
Normally constants are named in ALL_UPPER_CASE.
Can you call this `HTTP_STATUS` instead?

Reply:
Done